### PR TITLE
expose the readwrite section

### PR DESCRIPTION
**Pull request by Geal**
# Description

The readwrite section is part of the code generated by LLVM, and should be accessible if we want to move things around when compiling position independent code

### CODE REVIEW

**Comment by Geal** [comment]
while I'm at it, it would be just as useful to open access to the read section, should I update the PR with that?

File: lib/runtime-core/src/backend.rs [hunk 1]
```diff
@@ -243,6 +243,14 @@ pub trait RunnableModule: Send + Sync {
         None
     }
 
+    fn get_readwrite_section(&self) -> Option<&[u8]> {
```
Comment (Hywan) [inline lib/runtime-core/src/backend.rs]: Please add documentation here :-).

File: lib/runtime-core/src/backend.rs [hunk 2]
```diff
@@ -243,6 +243,14 @@ pub trait RunnableModule: Send + Sync {
...
+        None
+    }
+
+    fn get_read_section(&self) -> Option<&[u8]> {
```
Comment (Hywan) [inline lib/runtime-core/src/backend.rs]: That part needs documentation too.

**Comment by syrusakbary** [comment]
Closing PR since it was targeting the old Wasmer codebase and the codebase has changed widely

**Pull request closed without merging**